server/src/userManagement.mjs

```diff
@@ -228,11 +228,21 @@ function addUser(userId) {
     ws.on('pong', async hb => {
       heartbeat(ws)
     });
-    associateUserWithWs(''+userId, ws);
-    handleGroupMembership(''+userId)
-    ws.on('message', async message => {
-      messageHandler.handleMessage(message, ''+userId, ws);
-    });
+    if ((config.multiUserConnections == 'allow') || (config.multiUserConnections == 'warn') || (config.multiUserConnections == 'deny' && getWsForUser(userId) == undefined)) {
+      if (config.multiUserConnections == 'warn') {
+        logger.importantWarning(`WARNING: We do not support multiple websocket connections for a single user`)
```

Comment:
This can be a bit more descriptive. Something like

"Mock Firebolt was not written to support multiple connections for a single user. Some advanced use cases may result in unexpected behaviors."